.github/workflows/run_g_tests.yml

```diff
@@ -6,15 +6,39 @@ on:
 
 jobs:
   run_g_tests:
-    name: Run G Tests
-    runs-on: self-hosted
+    name: Run G Tests LV${{ matrix.version }}
+    runs-on: 
+      - self-hosted
+      - windows
+      - x64
+      - rdss-measlinkbot-win-10-lv64
+    strategy:
+      matrix:
+        version: ["2020", "2021", "2022", "2023"]
+      fail-fast: false
```

Comment:
For Python, the reasoning behind `fail-fast: false` was "Fail-fast skews the pass/fail ratio and seems to make pytest produce incomplete JUnit XML results." Does that apply here? I don't know how we are reporting/summarizing LV test results.

Reply:
I have removed `fail-fast` since it is not needed in this case. The tests are run from dedicated `run_tests.vi` which runs tests and displays the number of tests passed/failed along with failed tests reason if any in the same shell the this python script has been run.
https://github.com/ni/measurementlink-labview/actions/runs/7503475871/job/20428303932#step:4:17